tests/integration_tests/build/test_coverage.py

```diff
@@ -81,6 +81,13 @@ def test_coverage(test_fc_session_root_path, test_session_tmp_path):
     exclude_region = "'mod tests {'"
     target = "{}-unknown-linux-musl".format(platform.machine())
 
+    # Temporarily modify PATH so that "rustc" resolved to our wrapper script
+    # This is required as cargo kcov parses cargo's verbose output, and looking for lines starting with "Running `rustc":
+    # https://github.com/kennytm/cargo-kcov/blob/7a6bd499a1f9d4719a20cc9a37b6600fa1c9f68a/src/target_finder.rs#L44
+    # However, if we use the RUSTC_WRAPPER environment variable to make cargo run out script, the output will
+    # contain the lines "Running /firecracker/tests/rustc", and thus cargo kcov will fail to resolve the paths of the
+    # binaries compiled.
+    os.environ["PATH"] = "/firecracker/tests" + os.pathsep + os.environ["PATH"]
     cmd = (
```

Comment:
Did you try using `CARGO_TARGET_x86_64_unknown_linux_musl_RUSTFLAGS=-Clink-dead-code` as an alternative?

Reply:
Yes, those flags only affect the code that is actually compiled to run on the specified target, and thus also excludes proc macros and build scripts

Reply:
Actually it should be `CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_RUSTFLAGS=-Clink-dead-code` (uppercased), but if it's not going to apply by design then probably not worth trying again.

Reply:
I did try both! (if you use the lowercase option then cargo will tell you to use the uppercase one instead)